index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/4e655bdddb.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <h1 class="logo"><a href="#">SP</a></h1>
```

Comment:
- [ ] For schematic reasons, only one `h1` element should be in your project. Since you have it in the intro, please consider using fonts to style the logo :
- The main reason for this is that search engines use the h1 element to understand the primary topic of the page. If there are multiple h1 elements on a page, it can confuse search engines and make it more difficult for them to accurately determine the main topic of the page.

Reply:
I updated it as stated